app/views/admin/communities/analytics.haml

```diff
@@ -10,7 +10,11 @@
       %h3= t(".google_analytics_key")
     .row
       .col-6
-        = render :partial => "layouts/info_text", :locals => { :text => t(".google_analytics_key_info_text")}
+        - if APP_CONFIG.display_knowledge_base_articles
```

Comment:
Ahem, the "instructions_link" is a parameter to the translation of the text, so your code is not quite correct. But I'll do something and see what you think. :)